### PR TITLE
Create java-custom-metrics-micrometer.md

### PR DESCRIPTION
**Pull request by yotamloe**
# What changed
New doc for java custom metrics with micrometer
https://deploy-preview-1015--logz-docs.netlify.app/shipping/prometheus-sources/java-custom-metrics-micrometer.html

https://deploy-preview-1015--logz-docs.netlify.app/shipping/prometheus-sources/java-custom-metrics.html

Need to duplicate it and create the same page for micrometer 

[Github project](https://github.com/logzio/micrometer-registry-logzio)
[Maven](https://search.maven.org/artifact/io.logz.micrometer/micrometer-registry-logzio/1.0/jar)

### CODE REVIEW

**Review by YajB — COMMENTED** [review]
created 2nd doc: Micrometer for custom metrics.
@Doron-Bargo @yotamloe  - review changes, please

File: _source/logzio_collections/_prometheus-sources/java-custom-metrics-micrometer.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  - yberlinger
+shipping-tags:  
+  - prometheus
+---
+
+To send custom metrics from your Java application to Logz.io, you'll need to use a [Micrometer metrics](https://micrometer.io/) registry for dimensional metrics.
+
+This registry can be used by any application that uses Micrometer for recording metrics.
+
+## Usage:
```
Comment (YajB) [inline _source/logzio_collections/_prometheus-sources/java-custom-metrics-micrometer.md:26]: Does this section need further explanations? 
Why has the heading skipped from H2 to H4 ?

Comment (Doron-Bargo) [reply _source/logzio_collections/_prometheus-sources/java-custom-metrics-micrometer.md:26]: Maybe change the following sentences:
To send custom metrics from your Java application to Logz.io, you’ll need to use a Micrometer metrics registry for dimensional metrics.

This registry can be used by any application that uses Micrometer for recording metrics

To :
Micrometer (add link to micrometer site) provides a simple facade over the instrumentation clients for the most popular monitoring systems, allowing you to instrument your JVM-based application code without vendor lock-in. 
Logz.io's Micrometer metrics registry sends your custom metrics to your Logz.io account.

Comment (YajB) [reply _source/logzio_collections/_prometheus-sources/java-custom-metrics-micrometer.md:26]: @Doron-Bargo -
Is the suggested change relevant for both topics or only for `_source/logzio_collections/_prometheus-sources/java-custom-metrics-micrometer.md`?

**Review by YajB — APPROVED** [review]

**Pull request merged**